source/slang/slang-parser.cpp

```diff
@@ -7771,6 +7793,20 @@ namespace Slang
             parser->ReadToken(TokenType::Comma);
         }
 
+#define CASE(key, type) if (AdvanceIf(parser, #key)) { auto modifier = parser->astBuilder->create<type>(); \
+    modifier->location = getIntegerLiteralValue(listBuilder.find<GLSLLayoutModifier>()->valToken); listBuilder.add(modifier); } else
```

Comment:
Is "modifier->location" only for debugging purpose?

Reply:
modifier->location is for creating a modifier and tying a glsl location attribute to it